x-pack/filebeat/module/mysql/audit/_meta/fields.yml

```diff
@@ -0,0 +1,121 @@
+- name: audit
+  type: group
+  release: beta
+  default_field: false
+  description: >
+    Module for parsing MySQL Enterprise Audit Logs
+  fields:
+    - name: class
+      type: keyword
+      description: >
+        A string representing the event class. The class defines the type of event, when taken together with the event item that specifies the event subclass.
+
+    - name: connection_id
+      type: keyword
+      description: >
+        An integer representing the client connection identifier. This is the same as the value returned by the CONNECTION_ID() function within the session.
+
+    - name: id
+      type: keyword
+      description: >
+        An unsigned integer representing an event ID.
+
+    - name: connection_data.connection_type
+      type: keyword
+      description: >
+        The security state of the connection to the server. Permitted values are tcp/ip (TCP/IP connection established without encryption), ssl (TCP/IP connection established with encryption), socket (Unix socket file connection), named_pipe (Windows named pipe connection), and shared_memory (Windows shared memory connection).
+
+    - name: connection_data.status
+      type: long
+      description: >
+        An integer representing the command status: 0 for success, nonzero if an error occurred.
+
+    - name: connection_data.db
+      type: keyword
+      description: >
+        A string representing a database name. For connection_data, it is the default database. For table_access_data, it is the table database.
+
+    - name: connection_data.connection_attributes
+      type: flattened
+      description: >
+        Connection attributes that might be passed by different MySQL Clients.
+
+    - name: general_data.command
+      type: keyword
+      description: >
+        A string representing the type of instruction that generated the audit event, such as a command that the server received from a client.
+
+    - name: general_data.sql_command
+      type: keyword
+      description: >
+        A string that indicates the SQL statement type.
+
+    - name: general_data.query
+      type: keyword
+      description: >
+        A string representing the text of an SQL statement. The value can be empty. Long values may be truncated. The string, like the audit log file itself, is written using UTF-8 (up to 4 bytes per character), so the value may be the result of conversion.
+
+    - name: general_data.status
+      type: long
+      description: >
+        An integer representing the command status: 0 for success, nonzero if an error occurred. This is the same as the value of the mysql_errno() C API function.
+
+    - name: login.user
+      type: keyword
+      description: >
+        A string representing the text of an SQL statement. The value can be empty. Long values may be truncated. The string, like the audit log file itself, is written using UTF-8 (up to 4 bytes per character), so the value may be the result of conversion.
```

Comment:
I think this is the same description as the one from general_data.query.  Guessing it needs to be updated.